riva/proto/riva_asr.proto

```diff
@@ -393,6 +393,11 @@ message StreamingRecognizeResponse {
   RequestId id = 100;
 }
 
+message PipelineStates{
+  // Neural VAD probabilities
+  repeated float vad_probabilities = 1;
```

Comment:
How does user interpret repeated vad probabilities?

Reply:
These probabilities would help the clients if they want to know when the speech/silence started. Current usecase: ACE pipeline for barge-in support.